modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/WindowsAPIUtils.java

```diff
@@ -199,4 +199,12 @@ public static OAuth2TokenValidationService getOAuth2TokenValidationService() {
         return oAuth2TokenValidationService;
     }
 
+    public static void startTenantFlow(String userName) {
```

Comment:
yes.It was used in several places.

[calling this function just before a function return.] that issue is fixed from https://github.com/wso2/product-mdm/pull/156